runtime-light/stdlib/fork/fork-context.h

```diff
@@ -12,7 +12,7 @@
 #include "runtime-light/coroutine/task.h"
 #include "runtime-light/utils/concepts.h"
 
-constexpr int64_t INVALID_FORK_ID = -1;
+inline constexpr int64_t INVALID_FORK_ID = -1;
```

Comment:
Is inline make sense in such cases?

Reply:
Yep, it guarantees that we only have one instance of that value.